news/_posts/2022-01-05-multistore-configuration-forms.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Multistore configuration forms"
+subtitle: "How multistore configuration forms got better with PrestaShop 1.7.8"
+date: 2022-01-05 
+authors: [matthieu-rolland]
+icon: icon-puzzle
+tags: [multistore, contribute]
+---
+
+Did you know that PrestaShop has a cool feature called "multistore", which is the ability to manage multiple shops from the same back office ?
```

Comment:
The 1st sentence of the post is used as a summary inside the homepage https://build.prestashop.com/
so we need to write something that summarizes your article.

Suggestion:
> With the release of PrestaShop 1.7.8, we expanded PrestaShop multistore capabilities to Symfony forms to build a reusable form component to make pages multistore-compatible

Reply:
Also

> Did you know that PrestaShop has a cool feature called "multistore", which is the ability to manage multiple shops from the same back office ?

It sounds a bit like a teaser for a film? 😄  I think almost all Build readers know about this feature.